redislock.go

```diff
@@ -181,14 +221,14 @@ func (l *Lock) Release(ctx context.Context) error {
 		return ErrLockNotHeld
 	}
 
-	res, err := luaRelease.Run(ctx, l.client, []string{l.key}, l.value).Result()
+	res, err := luaRelease.Run(ctx, l.client, l.keys, l.value).Result()
 	if err == redis.Nil {
 		return ErrLockNotHeld
 	} else if err != nil {
 		return err
 	}
 
-	if i, ok := res.(int64); !ok || i != 1 {
+	if releaseCount := res.(int64); releaseCount != int64(len(l.keys)) {
```

Comment:
One more case when script could return true/false on its own.

Doesn't prevent us from merging and then polishing it ourselves.

Reply:
Same as above.